src/main/java/org/jvnet/hudson/test/JenkinsSessionRule.java

```diff
@@ -55,6 +55,15 @@
      */
     private int port;
 
+    /**
+     * Get the Jenkins home directory, which is consistent across restarts.
+     * Normally it will suffice to use {@link LocalData} to populate files.
```

Comment:
I am not actually sure if `@LocalData` works with `JenkinsSessionRule`.

Reply:
Ok I will check.

Reply:
From a quick test `@LocalData` does seem to work with `JenkinsSessionRule`.

Reply:
Ok looking a bit deeper, the behavior is not really what you would want, so I will remove the recommendation. `@LocalData` does function, but each time Jenkins restarts, the `@LocalData` replaces the contents of `JENKINS_HOME`, and using it at all causes a new `JENKINS_HOME` to be allocated by `HudsonHomeLoader`, leaving `JenkinsSessionRule.home` unused (and distinct from `Jenkins.root`).

Reply:
See 71560d6.